src/main/java/frc/team4909/robot/Robot.java

```diff
@@ -56,10 +64,19 @@ public void robotInit() {
     m_chooser.setDefaultOption("Default Auto", kDefaultAuto);
     m_chooser.addOption("My Auto", kCustomAuto);
     SmartDashboard.putData("Auto choices", m_chooser);
+
+    driverGamepad = new BionicF310(0, 0, 0.6);  //Creates new drivergamepad object
+    drivetrainsub = new DriveTrainSubsystem(); //Creates new drivetrain subsytem object
+    //photoelectricsensors = new PhotoElectricSensors();
+    
+    analogInput = new AnalogInput(0);
```

Comment:
The whole point of the issue was to move the IR code from general testing into the Intake Subsystem specific code. 

The goal is for the intake to have a `boolean isCargoAcquired()` method that returns false if the IR is returning a known voltage (representing the material on the other end of the intake) or true if the voltage differs. This should not be a basic equals but rather thresholded instead.